### PR TITLE
Add guidance for getting new components introduced to Primer

### PR DESCRIPTION
**Pull request by mperrotti**
These changes add a new page with information about how a component can be "on-ramped" into Primer from dotcom, or how a new component can be added directly to Primer.

### CODE REVIEW

File: content/guides/contribute/adding-new-components.mdx [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: Adding new components
+description: Guidance to help you get your components added to Primer. This primary audience for this guidance is GitHub staff.
```
Comment (langermank) [inline content/guides/contribute/adding-new-components.mdx:3]:
```suggestion
description: Guidance to help you get your components added to Primer. The primary audience for this guidance is GitHub staff.
```

File: content/guides/contribute/adding-new-components.mdx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+Once a component is upstreamed, it goes through a [component lifecycle maturity](/guides/component-lifecycle) process until it reaches the stable level.
+
+Once a component meets “Alpha” criteria it's considered safe to be use in production, but consumers can expect breaking changes as it matures to “Beta”. We aim to get all or most components to the “Stable” level of maturity criteria.
+
+## Adding components directly to Primer
+
+Although Primer has a high barrier of entry for introducing new components, contributions from everyone are welcome and encouraged. The contribution documentation and the Primer team members are available to guide new component contributions.
+
+Primer maintainers often lack the bandwidth to do the work for you, but will do our best to answer your questions and help get your contribution to the “Alpha” maturity level.
```
Comment (langermank) [inline content/guides/contribute/adding-new-components.mdx:92]:
```suggestion
Primer maintainers often lack the bandwidth to do the work for you, but will do their best to answer your questions and help get your contribution to the “Alpha” maturity level.
```

**Review by langermank — APPROVED** [review]
Small grammar suggestions

File: content/guides/contribute/adding-new-components.mdx [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+This guidance is intended for GitHub staff members who are introducing design patterns that are not yet supported by Primer components. Contributions from Primer consumers who don't work at GitHub are welcome and encouraged, but these steps assume the reader is working on a feature team at GitHub.
+
+### 1. Exists and is discoverable
+
+Your component should have:
+
+- Name
+- Description
+- At least 1 Storybook story
+- No aXe violations
```
Comment (mperrotti) [inline content/guides/contribute/adding-new-components.mdx:19]: This might get moved to "4. Is accessible"

**Pull request merged**